src/presolve.jl

```diff
@@ -590,6 +602,21 @@ function remove_empty_columns!(ps::PresolveData{T}) where {T}
     return nothing
 end
 
+"""
+    ensure_integer_bounds!(ps::PresolveData)
+
+Ensure all columns with integer variables having integer bounds.
+
+Called once at the very beginning of the presolve procedure.
+"""
+
+function ensure_integer_bounds!(ps::PresolveData{T}) where {T}
+    for j in 1:ps.pb0.nvar
+        ensure_integer_bounds!(ps, j)
+    end
+    return nothing
+end
+
```

Comment:
Once #7 is merged, I'd like to follow the convention therein:
* Introduce a `struct TightenIntegerBounds <: AbstractPresolveRule` (the name is only for illustration) with detailed documentation. Everything related to this should be in the same file.
* In the main presolve loop, use the calling convention `apply!(ps, rule, config)`